lib/core/src/lib/layout/components/header/header.component.html

```diff
@@ -27,7 +25,8 @@
     </a>
 
     <span
-        role="link"
```

Comment:
Is removing the `link` role correct? It was here before for some reason

Reply:
Yes that is a title with link and that need to provide heading role.

Reply:
But then the title won't have `link` role, can this cause new accessibility issue? Since it's a router link

Reply:
I think this link is working without link role too and i checked this is not introducing new accessibility issue.

Reply:
Link itself works but it won't be correctly announced by assistive technology, if link is not contain within `a` element it should have `link` role. Please check https://dequeuniversity.com/checklists/web/links, Axe Devtool is just a automated tool and it isn't able to detect all issues. You should always think yourself if your change will introduce new accessibility issue. Fixing one while creating new one on the way is not the right way...

Reply:
added the link role back.

Reply:
Okay now what about `heading` role? As far as I understood from the linked Jira it should be applied to the title right?